lib/venue.rb

```diff
@@ -1,2 +1,17 @@
 class Venue
-end
+   attr_reader :name, :capacity, :patrons
+
+   def initialize(name, capacity)
+     @name = name
+     @capacity = capacity
+     @patrons = []
+   end
+   
+   def add_patron(patron)
+     @patrons << patron
+   end
+
+   def yell_at_patrons
+     @patrons.map(&:upcase)
```

Comment:
This works! In the future, I'd prefer for you to use the syntax with `do/end` blocks as it will help you understand what's going on and be more flexible if you have to modify your code.